fpdf/line_break.py

```diff
@@ -474,52 +482,120 @@ def automatic_break(self, justify: bool):
                 self.hyphen_break_hint.k,
                 self.hyphen_break_hint.original_fragment_index,
                 self.hyphen_break_hint.original_character_index,
+                self.height,
             )
             return (
                 self.hyphen_break_hint.original_fragment_index,
                 self.hyphen_break_hint.original_character_index,
-                self.manual_break(justify),
+                self.manual_break(align),
             )
         self._apply_automatic_hint(self.space_break_hint)
         return (
             self.space_break_hint.original_fragment_index,
             self.space_break_hint.original_character_index,
-            self.manual_break(justify),
+            self.manual_break(align),
         )
 
 
 class MultiLineBreak:
     def __init__(
         self,
-        styled_text_fragments: Sequence,
-        justify: bool = False,
+        fragments: Sequence,
+        max_width: Union[float, callable],
+        margins: Sequence,
+        align: Align = Align.L,
         print_sh: bool = False,
         wrapmode: WrapMode = WrapMode.WORD,
+        line_height: float = 1.0,
+        skip_leading_spaces: bool = False,
     ):
-        self.styled_text_fragments = styled_text_fragments
-        self.justify = justify
+        """Accept text as Fragments, to be split into individual lines depending
+        on line width and text height.
+        Args:
+            fragments: A sequence of Fragment()s containing text.
+            max_width: Either a fixed width as float or a callback function
+                get_width(height). If a function, it gets called with the largest
+                height encountered on the current line, and must return the
+                applicable width for the line with the given height at the current
+                vertical position. The height is relevant in those cases where the
+                lateral boundaries of the enclosing TextRegion() are not vertical.
+            margins (sequence of floats): The extra clearance (usually FPDF.c_margin)
```

Comment:
It may be worth to specify how many values can be provided: 1? 2? 4?

Could it be meaningful to use `util.Padding` there, instead of a sequence of numbers with unspecified size?

Reply:
Technically, as many as you like... :wink:
In practise it is two, each of which is either `FPDF.c_margin` or zero (if padding is applied on the respective side). The numbers are in document units.

Reply:
Alright. This could be documented, and potentially checked at runtime

Reply:
I've made the docstring a bit clearer.